scripts/templates/table/table.ejs

```diff
@@ -1,21 +1,24 @@
+<%
+var title = [];
+var unit = [];
```

Comment:
Uhm... Not sure if I have understood this... Are you looping the data and setting variables in a template? Never do that. Templates are for render vars, the less logic, the better. If you need to organize the data before render it, use some method inside the component.